sound/soc/intel/boards/sof_sdw.c

```diff
@@ -870,6 +870,31 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		},
 		.dai_num = 1,
 	},
+	{
+		.part_id = 0x722,
+		.version_id = 3,
+		.dais = {
+			{
+				.direction = {true, true},
+				.dai_name = "rt722-sdca-aif1",
+				.dai_type = SOF_SDW_DAI_TYPE_JACK,
+				.dailink = {SDW_JACK_OUT_DAI_ID, SDW_JACK_IN_DAI_ID},
+			},
+			{
+				.direction = {true, false},
+				.dai_name = "rt722-sdca-aif2",
+				.dai_type = SOF_SDW_DAI_TYPE_AMP,
+				.dailink = {SDW_AMP_OUT_DAI_ID, SDW_UNUSED_DAI_ID},
```

Comment:
are you sure there's no amp feedback for speaker protection? That's worthy of a comment.

Reply:
> are you sure there's no amp feedback for speaker protection? That's worthy of a comment.

The rt722-sdca codec driver doesn't provide capture capability for "rt722-sdca-aif2" dai. So, we can't create a capture dai link for it. And so does rt712-sdca.